packages/manager/modules/pci/src/projects/onboarding/onboarding.html

```diff
@@ -0,0 +1,14 @@
+<div class="pci_projects_onboarding">
+    <div class="row">
+        <div class="col-md-6">
+            <h1 data-translate="pci_projects_onboarding_title"></h1>
+            <p data-translate="pci_projects_onboarding_description1"></p>
+            <p data-translate="pci_projects_onboarding_description2"></p>
+            <p data-translate="pci_projects_onboarding_description3"></p>
+            <oui-button class="mt-4" variant="primary">
```

Comment:
`data` prefix is used everywhere into this template file. I encourage you to stick with it.

```suggestion
            <oui-button class="mt-4" data-variant="primary">
```